device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O8V48/sai.profile

```diff
@@ -0,0 +1,4 @@
+SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_4700_8x400g_48x200g.xml
+SAI_DUMP_STORE_PATH=/var/log/mellanox/sdk-dumps
+SAI_DUMP_STORE_AMOUNT=10
+SAI_DUMP_MFT_CFG_PATH=/etc/sonic/mft/fwtrace_cfg
```

Comment:
We need to add store and forward for MSFT SKUs https://github.com/sonic-net/sonic-buildimage/blob/04bb0d34947037c182bf8e3b9818dd693f232822/device/mellanox/x86_64-mlnx_msn4700-r0/Mellanox-SN4700-O8C48/sai.profile#L5